modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -302,6 +301,24 @@ export class Tile3D {
     return this._distanceToCamera;
   }
 
+  /**
+   * Memory usage of tile on GPU
+   */
+  get gpuMemoryUsageInBytes(): number {
```

Comment:
I'd drop the getter and just precalculate this in the constructor and add it to a read-only field.